.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+# Github worflows for Ocean
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    name: Run
+    strategy:
+      # Disable `fail-fast` because we want the whole test suite to run even if one
+      # of the nigthly is broken
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-18.04 ]
+        dmd: [ '2.092.*', '2.093.*' ]
+        flavor: [ prod, devel ]
+        include:
+          - { os: ubuntu-18.04, dmd: '2.092.*', dist: bionic, coverage: 0, closure_check: false }
+          - { os: ubuntu-18.04, dmd: '2.093.*', dist: bionic, coverage: 1, closure_check: true  }
+
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
+    steps:
+
+    # Checkout this repository and its submodules
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Setup environment
+      run: |
+        echo "$(git config -f .gitmodules submodule.beaver.path)/bin" >> $GITHUB_PATH
+        echo "COV=${{ matrix.coverage }}" >> $GITHUB_ENV
+        echo "DIST=${{ matrix.dist }}" >> $GITHUB_ENV
```

Comment:
It should be possible to get this with `lsb_release -cs` without needing to include a separate `dist` parameter in the matrix config.  That might help avoid future errors where the `os` setting gets updated but the `dist` does not.

Reply:
I actually want to get rid of this and move the work from beaver to the Github action directly.
Currently the Github action runs in a container, which itself spawns a container. That's quite superfluous.
Plus, if we want to test on other platforms (e.g. OSX) we can't rely on beaver / `lsb_release`.